smac/utils/util_funcs.py

```diff
@@ -16,6 +16,11 @@ def get_types(config_space, instance_features=None):
             types[i] = n_cats
             bounds[i] = (int(n_cats), np.nan)
 
+        elif isinstance(param, (OrdinalHyperparameter)):
+            n_cats = len(param.sequence)
```

Comment:
I don't know the details of the new rf API. But it is important that ordinals are not handled in the same way as categorical, because they have an ordering and categorical have not. So, the split on such feature can be different.

Without looking into the API docu, I would expect something along these lines:
`bounds[i] = (0.0, len(param.choices)-1)`
`types[i]`  should still be set to 0 since it is not a categorical feature.